src/components/material/material-buttons/online/MaterialButtonOnlineExternal.tsx

```diff
@@ -44,10 +47,27 @@ const MaterialButtonOnlineExternal: FC<MaterialButtonOnlineExternalProps> = ({
     }
   }, [data, error, translatedUrl, urlWasTranslated]);
 
+  const label = (sourceName: string, materialTypes: string[]) => {
+    if (sourceName.includes("ereol")) {
+      return `${t("goToText")} ereolen`;
+    }
+    if (sourceName.includes("filmstriben")) {
+      return `${t("goToText")} filmstriben`;
+    }
```

Comment:
You should make use of the placeholder functionality in the new text functionality. See readme ;)

Reply:
Okay I shall :3 but those changes you introduced are not a part of this PR 🙈 But I guess that's alright, right? I'll just use it "as if" it already existed here.

Reply:
Oh, I found it now... I think maybe you should mention the document in the readme file as well. That's where I looked 🙈

Reply:
Don't get anything of what you wrote but glad that you have fixed it.